src/self-model.scm

```diff
@@ -583,6 +583,7 @@
 (DefineLink
 	(DefinedPredicate "look at person")
 	(SequentialAnd
+		(DefinedPredicate "Time to change gaze")
```

Comment:
This also looks wrong to me. The time-to-change-gaze should be inside of openpsi, which is deciding what to do next.  By contrast, "look at person" is supposed to be a direct, immediate action, carried out right away.    That is, openpsi should be deciding if and when it is time to change the gaze, and if it is, then the look should do that, instantly.

Also -- you seem to be confusing "looking" which turns the whole head, with "gazing" which moves only the eyes. 

There is also a third problem, below: "glance" is supposed to be a momentary gaze: the eyes turn in that direction only breifly, say 1/10th to 1/2 of a second, and then turn back to their original target.  This whole thing is starting to feel like spaghetti code.